node-tests/addon-test.js

```diff
@@ -17,6 +17,16 @@ describe('ember-cli-babel', function() {
     });
   });
 
+  describe('Deprecate ember-cli-babel 5', function() {
+    it('should always print deprecaiton', function() {
+      var deprecationMessages = this.ui.output.split('\n').filter(function(line) {
+        return line.indexOf('Write actual deprecation') !== -1;
```

Comment:
(Supported until 3.4 of Ember-CLI)

Reply:
"ember-cli-babel 5.x has been deprecated. Please upgrade to at least ember-cli-babel 6.6.0" cc @raytiley